app/src/main/java/nerd/tuxmobil/fahrplan/congress/favorites/StarredListAdapter.kt

```diff
@@ -51,15 +51,20 @@ class StarredListAdapter internal constructor(
 
             title.textOrHide = session.title
             subtitle.textOrHide = session.subtitle
+            subtitle.contentDescription = session.getSubtitleContentDescription(context)
             speakers.textOrHide = session.formattedSpeakers
+            speakers.contentDescription = session.getSpeakersContentDescription(context)
             lang.textOrHide = session.lang
             lang.contentDescription = session.getLanguageContentDescription(context)
             day.isVisible = false
             val timeText = DateFormatter.newInstance(useDeviceTimeZone).getFormattedTime(session.dateUTC, session.timeZoneOffset)
             time.textOrHide = timeText
+            time.contentDescription = session.getStartTimeContentDescription(context, timeText)
             room.textOrHide = session.room
-            val durationText = context.getString(R.string.session_duration, session.duration)
+            room.contentDescription = session.getRoomNameContentDescription(context)
+            val durationText = context.getString(R.string.session_list_item_duration_text, session.duration)
             duration.textOrHide = durationText
+            duration.contentDescription = session.getDurationContentDescription(context)
             video.isVisible = false
             noVideo.isVisible = false
             withoutVideoRecording.isVisible = session.recordingOptOut
```

Comment:
Maybe some newlines help here to make it more readable.

Reply:
:heavy_check_mark: Updated in 380170d8d2e50c3fd8ee24d0ae5e7304e4fb4bcf.